Add IEChooser.exe info (#1315)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/ie-mode/index.md

    *   Select `F12`.  
    *   Hover anywhere, open a contextual menu \(right-click\), and choose **Inspect element**.  For more information about how to use those tools, navigate to [Using the F12 developer tools][PreviousVersionsWindowsInternetExplorerDeveloperSamplesbg182326].  

> If Internet Explorer 11 Standalone is not available (such as on Windows 11), use IEChooser to launch Internet Explorer DevTools. Perform the following steps:

Reviewer: ```suggestion
If Internet Explorer 11 is not available, such as on Windows 11, you can use IEChooser to launch the Internet Explorer DevTools to debug the content of your IE mode tabs. To use IEChooser, perform the following steps.
```

1.  Open IEChooser.

>     1. Press the `Windows logo key` + `R` to open the Run dialog box. 

Reviewer: ```suggestion
    1. Open the Run dialog box. For example, select the `Windows logo key` + `R`.
```

>     2. In the Run dialog box, enter `%systemroot%\system32\f12\IEChooser.exe`.

Reviewer: ```suggestion
    2. Enter `%systemroot%\system32\f12\IEChooser.exe`, and then select **Ok**.
```

2.  In IEChooser, select the entry for the IE mode tab.

> An Internet Explorer DevTools window opens to debug the content of the IE mode tab.

Reviewer: ```suggestion
```
